### PR TITLE
Added new phishing domains

### PR DESCRIPTION
**Pull request by anishshandilya**
Added new phishing domains

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -966,6 +966,78 @@
...
+    "blur-newairdrop.com",
+    "blurprize.com",
+    "blur.expert",
+    "crypto-blur.site",
+    "blur-nft.info",
+    "vip-pudgypenguins.com",
+    "aptos-drops.com",
+    "openseaprotocol.com",
+    "rewarddoodles.com",
+    "zapper-frontend-next-8h1ycrhu9-zapper-fi.vercel.app",
```
Comment (ktian94) [inline src/config.json]: Remove these

**Review by ktian94 — CHANGES_REQUESTED** [review]

File: src/config.json [hunk 2]
```diff
@@ -966,6 +966,78 @@
...
+    "claims-sui.com",
+    "blursea.io",
+    "web3-privacy.com",
+    "unlocksolana.org",
+    "solanajobs.xyz",
+    "solanaludo.com",
+    "darksolana.com",
+    "aptools.xyz",
+    "gitcoinmint.site",
+    "zapper-frontend-next-4uip4z44q-zapper-fi.vercel.app",
```
Comment (ktian94) [inline src/config.json]: Rm

File: src/config.json [hunk 3]
```diff
@@ -966,6 +966,78 @@
...
+    "blur.tips",
+    "blur-coin.net",
+    "tooksrare.co",
+    "doodlesokx.one",
+    "blurmarkets.website",
+    "ms-blur.site",
+    "suicoin.info",
+    "www-x2y2.io",
+    "revoke-pudgypenguins.com",
+    "zapper-frontend-next-o41khhhh7-zapper-fi.vercel.app",
```
Comment (ktian94) [inline src/config.json]: Rm

File: src/config.json [hunk 4]
```diff
@@ -966,6 +966,78 @@
...
+    "theblurswap.io",
+    "blurapps.org",
+    "sui-tokenclaim.com",
+    "claims-sui.com",
+    "blursea.io",
+    "web3-privacy.com",
+    "unlocksolana.org",
+    "solanajobs.xyz",
+    "solanaludo.com",
+    "darksolana.com",
```
Comment (ktian94) [inline src/config.json]: Rm

File: src/config.json [hunk 5]
```diff
@@ -513,6 +513,69 @@
...
+    "theblurswap.io",
+    "blurapps.org",
+    "sui-tokenclaim.com",
+    "claims-sui.com",
+    "blursea.io",
+    "web3-privacy.com",
+    "unlocksolana.org",
+    "solanajobs.xyz",
+    "solanaludo.com",
+    "aptools.xyz",
```
Comment (ktian94) [inline src/config.json:546]: remove

**Review by ktian94 — CHANGES_REQUESTED** [review]

**Review by legobeat — CHANGES_REQUESTED** [review]
@anishshandilya Please rebase on `main` with your commits squashed, or reopen with a cleaner history.

Thanks.

**Pull request closed without merging**